### PR TITLE
views: show permissions associated to a token

### LINKED ISSUE #139: oauth2server: scopes / permissions given to an OAuth2 application / consumer cannot be reviewed?
It seems that user doesn't have a way to review the scopes / permissions given to an OAuth2 application after the initial authorization request (the screen where you can select to authorize the application or not).

For inspiration, here is how GitHub and Google have done their authorized OAuth application review pages:

https://github.com/settings/connections/applications/
https://github.com/settings/connections/applications/98387d50437fdf43d9da
oauth_scopes_review_example_github

https://myaccount.google.com/permissions
oauth_scopes_review_example_google

### LINKED ISSUE #147: examples: integrate consumer example application

### PR DESCRIPTION
**Pull request by hachreak**
* Adds a view to see which authorization is garanted to a
  application. (closes #139)

* Adds a client application as example. (closes #147)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
need review.. test are [passing](https://travis-ci.org/hachreak/invenio-oauth2server/builds/263052923)

**Comment by lnielsen** [comment]
Screenshot?

**Comment by hachreak** [comment]
auth

**Comment by hachreak** [comment]
ping @lnielsen

**Comment by hachreak** [comment]
@lnielsen changes done! :+1:

File: invenio_oauth2server/templates/invenio_oauth2server/settings/token_permission_view.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2014, 2016, 2017 CERN.
```
Comment (nharraud) [inline invenio_oauth2server/templates/invenio_oauth2server/settings/token_permission_view.html]: This looks like a new file. The copyright go in the past.

**Comment by lnielsen** [comment]
Rebased an integrated in #154. Once #154 is merged this PR should also show as merged.

**Pull request merged**